packages/ragbits-core/src/ragbits/core/prompt/base.py

```diff
@@ -37,6 +37,15 @@ def output_schema(self) -> Optional[Dict | Type[BaseModel]]:
         """
         return None
 
+    def list_images(self) -> list[dict]:
```

Comment:
I believe this should simply return a list of images, without the return format being coupled with the complicated LLM conversation format.

This is part of our public API and a method that our users will be expected to implement whenever they implement custom prompt sources. Is should be as simple as possible.

Reply:
done